hack/verify-golangci-lint.sh

```diff
@@ -21,15 +21,24 @@ set -o pipefail
 VERSION=v1.42.0
 URL_BASE=https://raw.githubusercontent.com/golangci/golangci-lint
 URL=$URL_BASE/$VERSION/install.sh
+INSTALL_CHECKSUM=294771225087ee48c8e0a45a99ac82ed8f9c6e9d384e692ab201986479c8594f
```

Comment:
do we need to generate this sha when we update the version or where do we find this? (did. quick look in the repo and did not find it)

sorry for the delay I was in vacation

Reply:
You would only need to update the check_sum if the install.sh script is updated or changed. For example I updated the version to 1.43 but did not need to update the checksum as the install script did not change.